docs/fundamentals/code-analysis/style-rules/ide0130.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "IDE0130: Namespace does not match folder structure"
+description: "Learn about code analysis rule IDE0130: Namespace does not match folder structure"
+ms.date: 12/05/2022
+ms.topic: reference
+f1_keywords:
+- IDE0130
+helpviewer_keywords:
+- IDE0130
+author: gewarren
+ms.author: gewarren
+dev_langs:
+- CSharp
+- VB
+---
+# Namespace does not match folder structure (IDE0130)
+
+| Property                 | Value                                                  |
+| ------------------------ | ------------------------------------------------------ |
+| **Rule ID**              | IDE0130                                                |
+| **Title**                | Namespace does not match folder structure              |
+| **Category**             | Style                                                  |
+| **Subcategory**          | Language rules (Namespace naming preferences)          |
+| **Applicable languages** | C# and Visual Basic                                    |
+| **Options**              | `dotnet_style_namespace_match_folder`                  |
+
+## Overview
+
+This style rule uses the folder structure of the project to enforce namespace naming requirements.
+
+## Options
+
+Options specify the behavior that you want the rule to enforce. For information about configuring options, see [Option format](language-rules.md#option-format).
+
+### dotnet_style_namespace_match_folder
+
+| Property                 | Value                                                | Description                                        |
+| ------------------------ | ---------------------------------------------------- | -------------------------------------------------- |
+| **Option name**          | dotnet_style_namespace_match_folder                  |                                                    |
+| **Option values**        | `true`                                               | Prefer namespace naming to match folder structure. |
+|                          | `false`                                              | Disables the rule.                                 |
+| **Default option value** | `false`                                              |                                                    |
```

Comment:
Isn't the default value `true`?

https://github.com/dotnet/roslyn/blob/19704e6876c4a5a88267af5cf83a4bfe6287d356/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/IdeCodeStyleOptions.cs#L64

Reply:
Oh yes! I forgot to update here. It's true with severity suggestion

Reply:
Do we not supply the default severity somewhere?